pkg/plugins/golang/v2/api.go

```diff
@@ -94,27 +92,29 @@ After the scaffold is written, api will run make on the project.
 }
 
 func (p *createAPISubcommand) BindFlags(fs *pflag.FlagSet) {
-	fs.BoolVar(&p.runMake, "make", true, "if true, run make after generating files")
+	p.options = &goPlugin.Options{CRDVersion: "v1beta1"}
+	fs.StringVar(&p.options.Group, "group", "", "resource Group")
```

Comment:
```suggestion
	fs.StringVar(&p.options.Group, "group", "", "resource Group")
	// Since domain is used in GVK comparison, only set domain if using project version 3+.
	if p.config.IsV3() {
		p.options.Domain = p.config.Domain
	}
```

Reply:
@estroz your suggestion here is not clear for me. 
1 - Why we are using the domain to compare GKV? Could you please link it?
2 - Then, we are implementing the interface/binding the flags. Why we should do that? we have none `domain` option for an api and add this flag ough to not be part of this PR.